docs/index.rst

```diff
@@ -98,28 +98,65 @@ The Drivers
 
 Drivers for NWP components are available as top-level CLI modes and API modules.
 
+Provided with a valid UW YAML configuration file, and CLI arguments when required, ``uw`` can prepare a fully provisioned run directory and execute a component either directly on the current system, or via a batch job submitted to an HPC scheduler.
+
+Each driver produces a list of available ``TASK`` arguments from its CLI ``--help, -h`` flag (shown
+below). The ``provisioned_run_directory`` will do everything except ``run`` the executable, but any
+of the tasks may be requested and only the steps required to produce that task will be performed.
+
+Over time, we'll add many other drivers to support a variety of UFS components from pre-processing to post-processing, along with many data assimilation components.
```

Comment:
This block moved here with slight modifications when I made two subsections here for UFS vs MPAS drivers.